packages/js/dependency-extraction-webpack-plugin/package.json

```diff
@@ -25,7 +25,7 @@
 		"assets/"
 	],
 	"dependencies": {
-		"@wordpress/dependency-extraction-webpack-plugin": "^3.7.0"
+		"@wordpress/dependency-extraction-webpack-plugin": "6.16.0"
```

Comment:
There is going to be a breaking change, starting https://github.com/WordPress/gutenberg/blob/trunk/packages/dependency-extraction-webpack-plugin/CHANGELOG.md#400-2022-08-24 there were three breaking changes entries.